app/routes/book_routes.py

```diff
@@ -23,8 +23,30 @@ def create_book():
 
 @books_bp.get("")
 def get_all_books():
-    query = db.select(Book).order_by(Book.id)
-    books = db.session.scalars(query)
+    query = db.select(Book)
```

Comment:
I'm assuming this will be discussed in the text, but maybe a comment like
```suggestion
    # create a basic select query without any filtering
    query = db.select(Book)
```
could be helpful?